pyuvsim/data/telescope_config_6ant_freqbuf.yaml

```diff
@@ -0,0 +1,7 @@
+Nants: 6
+beam_paths:
+  0: ../data/HERA_NicCST.uvbeam
+telescope_location: (-30.721527777777911, 21.428305555555557, 1073.0000000093132)
+telescope_name: HERA
+select:
+  freq_buffer: 1000000  # 1 MHz either side of simulated frequencies.
```

Comment:
This new option needs to be documented in our Sphinx docs, e.g. here: https://pyuvsim.readthedocs.io/en/latest/parameter_files.html